lib/mutant/matcher/method/instance.rb

```diff
@@ -12,10 +12,12 @@ class Instance < self
         # @param [UnboundMethod] method
         #
         # @return [Matcher::Method::Instance]
+        #
+        # :reek:ManualDispatch
         def self.new(scope, target_method)
           name = target_method.name
           evaluator =
-            if scope.include?(Memoizable) && scope.memoized?(name)
+            if scope.respond_to?(:memoized?) && scope.memoized?(name)
```

Comment:
Given mutant has a wide user base, and `memoized?` is a rellative common method name, should we test for both `include?(Memoizable) && respond_to?(:memoized?)``

Reply:
@mbj I wasn't able to kill that without creating a completely synthetic test case.

Reply:
@dkubb I'm fine with a synthetic test case. This class has verbose unit test coverage, adding this case should be possible. I'm fine to do such a fixup to this commit.